src/NuGet.Core/NuGet.CommandLine.XPlat/NuGet.CommandLine.XPlat.csproj

```diff
@@ -21,7 +21,8 @@
     <PackageReference Include="System.Runtime.InteropServices.RuntimeInformation" Version="4.0.0" />
     <PackageReference Include="Microsoft.Extensions.CommandLineUtils" Version="1.0.1" />
     <PackageReference Include="System.Runtime.Serialization.Primitives" Version="4.1.1" />
-    <PackageReference Include="Microsoft.Build.Runtime" Version="15.1.548" />
+    <PackageReference Include="Microsoft.Build.Runtime" Version="15.1.1012" />
+    <PackageReference Include="System.Collections.Immutable" Version="1.3.1" />
```

Comment:
nope. MSBuild.Runtime needs this now and was throwing otherwise. I think its a bug on their end.